lab_6_pipeline/pipeline.py

```diff
@@ -189,6 +303,10 @@ def _count_frequencies(self, article: Article) -> dict[str, int]:
         Returns:
             dict[str, int]: POS frequencies
         """
+        pos_dict = {}
+        for pos_tag in self.analyzer.from_conllu(article).get('upos'):
```

Comment:
do you get upos count for all text? check in the resulting meta files if pos frequencies are collected correctly

Reply:
Yes, there is upos count for all text